manager/state/raft/raft.go

```diff
@@ -511,6 +513,15 @@ func (n *Node) stop() {
 	n.stopMu.Lock()
 	defer n.stopMu.Unlock()
 
+	if n.isLeader() && len(n.cluster.Members()) > 1 {
+		ctx, cancel := context.WithTimeout(context.Background(), transferLeadershipTimeout)
+		err := n.transferLeadership(ctx)
```

Comment:
I don't understand how this code can work. To send the leadership transfer message, the raft state machine puts it in a `Ready` message that's received in `Run`. But this `Ready` loop never runs after calling `stop`.